### ISSUE TITLE
Inline Databases

### ISSUE THREAD

**Issue by ghost** [RafaelGB/obsidian-db-folder#1064]
Could you please add a feature for embedding databases inside of notes? This would actually be a game changer.

**Comment by ccc-dvansevenant** [comment]
> Could you please add a feature for embedding databases inside of notes? This would actually be a game changer.

Like this?

https://rafaelgb.github.io/obsidian-db-folder/features/Displaying%20data/

Embedding Database

![image](https://github.com/user-attachments/assets/09beccb1-6b86-4021-aa5c-5f8733c65728)

**Comment by ghost** [comment]
> > Could you please add a feature for embedding databases inside of notes? This would actually be a game changer.
> 
> Like this?
> 
> https://rafaelgb.github.io/obsidian-db-folder/features/Displaying%20data/
> 
> Embedding Database
> 
> ![image](https://github.com/user-attachments/assets/09beccb1-6b86-4021-aa5c-5f8733c65728)
> 

No, this just embeds the Dataview table—I mean, the interactive database folder table with all of its features. Similar to what you would do in Notion.

**Comment by ccc-dvansevenant** [comment]
> No, this just embeds the Dataview table—I mean, the interactive database folder table with all of its features. Similar to what you would do in Notion.

Ah. Thanks for the clarification.

It'll be interesting to see if that can be supported.

(I'm just finally digging in to DB Folder after having it sitting in my vault for a couple years)
